### ISSUE TITLE
More than 5 labels

### ISSUE THREAD

**Issue by mamaririnana08** [mike-kfed/roundcube-thunderbird_labels#62]
Hello, 

Is it possible to add more than 5 labels? And to modify the colors which are put?

**Comment by mike-kfed** [comment]
technically the code is on the way to allow any amount of labels, but since roundcube mangles the imap labelnames I stopped development in this direction, because it wont work right without patching roundcube itself.
changing the colors is possible in the `path/to/thunderbird_labels/skins/$skin/tb_label.css`.

**Comment by azmeuk** [comment]
Hi. @mike-kfed per chance, have you reported this issue upstream to roundcube?

**Comment by mike-kfed** [comment]
Yes I did, there's a forever-open issue about imap tags (see [their](https://github.com/roundcube/roundcubemail/) github issue 4986) where I explained the problem in one of the responses.

**Comment by mike-kfed** [comment]
related to #43

**Comment by Capercorn** [comment]
Hi, is there any news about the problem to add more than 5 labels?

**Comment by mike-kfed** [comment]
due to the fact that roundcube/roundcubemail#4986 is not resolved yet, it's not possible to implement this in a meaningful way
